### DISCUSSION TITLE
Issue in Bonsai

### DISCUSSION THREAD

**Discussion by arpi2203** [bonsai-rx/bonsai#1689] [Q&A]
Hi, Im using this code in Bonsai -c:v hevc_nvenc -pix_fmt yuv420p -preset p4 -tune hq -bf:v 0 -2pass 0 -gpu 0 -qp 30 -g 300 for a 4hrs recording but the memory consumption is too high and encoding collapses. Could anyone please tell me how to solve this so that it comsumes less memory and video can be recorded for 4hrs or more? Thanks

**Comment by bruno-f-cruz** [comment]
Hi,
Please read the guidelines to submit posts at #864. Specifically, please modify your post tittle and add the workflow.
Regarding your specific problem, the solution is likely to use a different codec, I am not familiar with the one you are using, but you are likely to be able to find a more efficient one around the web (e.g. https://www.ffmpeg.org/ffmpeg-codecs.html). Alternatively, you could save the video as RAW, decrease the frame rate or resolution.

B

**Comment by arpi2203** [comment]
Hi, Thanks for your response. I have reposted the question in a detailed
way following the guidelines. Could you please have a look and provide me
with some insights?
Thanks
Arpita

**Reply by bruno-f-cruz** [reply]
Hey! Please modify the title of the post. Also I believe I answered the question you asked?
